scripts/verify-merge-function-update.sh

```diff
@@ -1,48 +1,35 @@
 #!/usr/bin/env bash
+# Script makes sure that changes to the ParserProject struct and merge functions happen concurrently.
```

Comment:
I would rephrase this slightly: "Script to verify that if the ParserProject struct is updated, its merge functions are also updated."